Domain.FSharp/scratchpad.fsx

```diff
@@ -0,0 +1,43 @@
+#r "netstandard"
+
+open System
+open System.Text.RegularExpressions
+
+type TrackingId(value: Guid) =
+    member x.Value = value
+
+let m = TrackingId(Guid.NewGuid())
+
+m.Value
+
+type UnLocode(value: string) =
+
+    do
+        if value = null then raise (ArgumentNullException("value"))
```

Comment:
https://latkin.org/blog/2015/05/18/null-checking-considerations-in-f-its-harder-than-you-think/

Reply:
It's scratchpad. Please do look at that file, it's only for the interactive then I copy the code to fs files